dashboard/test/ui/runner.rb

```diff
@@ -641,7 +641,7 @@ def cucumber_arguments_for_browser(browser, options)
   arguments += skip_tag('@chrome_before_62') if browser['browserName'] != 'chrome' || browser['version'].to_i == 0 || browser['version'].to_i >= 62
   # browser version 0 implies the latest version.
   arguments += skip_tag('@no_older_chrome') if browser['browserName'] == 'chrome' && (browser['version'].to_i != 0 && browser['version'].to_i <= 67)
-  arguments += skip_tag('@no_safari_yosemite') if browser['browserName'] == 'Safari' && browser['platform'] == 'OS X 10.10'
+  arguments += skip_tag('@no_safari') if browser['browserName'] == 'Safari'
```

Comment:
I think this is going to make `@no_safari` start skipping iPad and iPhone. is this intended?

Reply:
`'safari' != 'Safari'` but I agree it's a little confusing, I can rework this line to check `browser['name']` directly instead.

Reply:
Good clarification, I missed that. This check looks better, thanks!